include/irr/asset/IRenderpassIndependentPipeline.h

```diff
@@ -182,7 +203,7 @@ struct SColorAttachmentBlendParams
     uint32_t blendEnable : 1;
     uint32_t srcColorFactor : 5;
     uint32_t dstColorFactor : 5;
-    uint32_t colorBlendOp : 2;
+    uint32_t colorBlendOp : 3;
```

Comment:
you need more bits for the future blend ops

Reply:
resolved